devito/propagator.py

```diff
@@ -178,8 +212,17 @@ def generate_space_loops_blocking(self, loop_body):
         ivdep = True
         remainder = False
         orig_loop_body = loop_body
+
+        inner_most_dim_passed = False  # used in oder to avoid at inner most dimension
```

Comment:
The flag `ivdep` two lines above serves the same purpose. Maybe use the same flag for both the things?